index.html

```diff
@@ -0,0 +1,17 @@
+<!DOCTYPE html>
+<html lang="en">
+
+<head>
+    <meta charset="UTF-8">
+    <meta http-equiv="X-UA-Compatible" content="IE=edge">
+    <meta name="viewport" content="width=device-width, initial-scale=1.0">
+    <title>Hello Microverse</title>
+    <link rel="stylesheet" href="styles.css">
+</head>
+
+<body>
+    <h1 class="h1">Hello Microverse!</h1>
+
+</body>
```

Comment:
- [ ] Keep doing the great work 👍

Reply:
@iambenkis Thank you for the review and feedback on the project. I will make the necessary changes to the Readme file as requested and will ensure that all commits follow the recommended format of a capitalized first letter.